Exercises-1-Basic/src/exercises/Number10Shape6.java

```diff
@@ -0,0 +1,27 @@
+package exercises;
+
+/*
+  iii)
+	
+
+     *
+    **
+   ***
+  ****
+ *****
+
+ */
+public class Number10Shape6 {
+
+    public static void main(String[] args) {
+	int number = 1;
+	while (number <= 5) {
+	    for (int i = 1; i <= 5 - number; i++)
+		System.out.print(" ");
```

Comment:
```suggestion
	    for (int i = 1; i <= 5 - number; i++) {
		System.out.print(" ");
	    }
```
The body of for loop should be inside of {}

Reply:
Done. Please check new file Number5Shape6